<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialogue</class>
 <widget class="QDialog" name="SettingsDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>220</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="hLayout_StyleSetting">
       <item>
        <widget class="QLineEdit" name="lineEdit_StylePath">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>QSS Style Path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_BrowseStyle">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_StartupWallet">
       <property name="title">
        <string>Startup Wallet</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,10">
        <item>
         <layout class="QHBoxLayout" name="hLayout_StartupWalletPath">
          <item>
           <widget class="QLineEdit" name="lineEdit_StartupWalletPath">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Startup Wallet Path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_BrowseStartupWallet">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="vLayout_EncryptionConfig">
          <item>
           <widget class="QComboBox" name="comboBox_EncryptionMode">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>No Encryption</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>AES 128</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>AES 192</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>AES 256</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_KeyMode">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>SHA-256 Digest</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direct Input</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_SaveSettings">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Save Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
